### DISCUSSION TITLE
Zotac Store ERROR 403. Possible captcha issue?

### DISCUSSION THREAD

**Discussion by jiglesiasc** [jef/streetmerchant#2523] [Help] [answered]
Hello everybody,

The bot is returning STATUS CODE ERROR 403 for Zotac Store, which I think is because Zotac Store has now a very strict captcha system with those annoying picture selection captcha and after a few minutes the site requires a new captcha to be solved. Of course the bot has no idea what to do with that.

Is there a way to bypass the captcha or the bot to ask me to solve the captcha, similar to what the bot already does with the discord server, but in this case using those pictures??

**Comment by ArmaanCPro** [comment, marked as answer]
I believe the best way would be to go into Discord, setup a webhook, and use the ATC button whenever it is in stock. Some stores are just too hard to solve a CAPTCHA for. Or, read this https://github.com/jef/streetmerchant/discussions/2554. I hope @jef adds this in the future.

**Reply by jiglesiasc** [reply]
Interesting. I wonder how good is 2captcha by solving rechaptcha which is the captcha system on Zotac Store. Even I as a human fail from time to time when solving recaptcha. It would be really nice to have the option to automatically solve captchas. Though the price is a bit too expensive. $3 / 1000 recaptchas. I think I need to solve around 2000 recaptcha to bot Zotac 24h
